source/includes/_feature-analyzing.md

```diff
@@ -220,7 +214,22 @@ POST datasets/{id}/multitables/ HTTP/1.1
 Location: datasets/{id}/multitables/3/
 ```
 
-...or GET an individual multitable:
+Analyses as described above are truly multidimensional; when you add another variable, the resulting cube obtains another dimension. Sometimes, however, you want to compare analyses side by side, typically looking at several (even *all*) variables against a common set of conditioning variables. For example, you might nominate "Gender", "Age", and "Race" as the conditioning variables and cross every other variable with those, in order to quickly discover common correlations.
+
+Multi-table definitions mainly provide a `template` member that clients can use to construct a valid query with the variable(s) of interest.
+
+Crunch provides a separate catalog where you can define and manage these common sets of variables. Like most catalogs, you can [GET it](#multitables162) to see which multitables are defined.
+
+
+
+#### Template query
+
+A multitable is a set of queries that form groups of ‘columns’ for different later chosen ‘row’ variables.
+It is defined by a **name** and a **template**. At minimum the template must contain a `query` fragment:
```

Comment:
_It turns out_ that `variable` is being required by the validator: https://www.pivotaltracker.com/story/show/137627897/comments/161004753